ga4gh/frontend.py

```diff
@@ -9,15 +9,19 @@
 
 import os
 import traceback
+import datetime
 
 import flask
 import flask.ext.api as api
```

Comment:
This particular import is used, but if we want to catch them more generally we should incorporate flake8 or something else (pyilnt?) into the build.  File an issue for that if you wish.